packages/typegpu/tests/std/numeric/div.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, expect, it } from 'vitest';
+import {
+  vec2f,
+  vec2i,
+  vec2u,
+  vec3f,
+  vec3i,
+  vec3u,
+  vec4f,
+  vec4i,
+  vec4u,
+} from '../../../src/data/vector.ts';
+import { div } from '../../../src/std/numeric.ts';
```

Comment:
Let's change these to imports of entry files, since we're testing the public API surface.
```suggestion
import * as d from '../../../src/data/index.ts';
import { div } from '../../../src/std/index.ts';
```